src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json

```
@@ -141,7 +141,7 @@
       "columnContextOpenOrder": "Bestellung öffnen",
       "emptyTitle": "Noch keine Bestellungen",
       "emptySearchTitle": "Keine Suchergebnisse",
-      "emptySubline": "Verwalte die Bestellungen dieses Kunden."
+      "emptySubline": "Hier können Sie die Bestellungen eines Kunden verwalten"
```

```suggestion
      "emptySubline": "Hier kannst Du die Bestellungen Deiner Kunden verwalten"
```
Wir nutzen im Admin "Du" als Anrede, nur in der Storefront wird "gesiezt".